src/server/api/routes/api-router.js

```diff
@@ -45,9 +46,15 @@ router.use('/topics', TopicsRouter);
 
 router.use('/skills', skillsRouter);
 
-router.use('/events', eventsRouter);
 router.use('/groups', groupsRouter);
 
+router.use('/feedbacks', feedbacksRouter);
+
+router.use('/events', eventsRouter);
```

Comment:
Why did you move this?

Reply:
Hi Toni, I moved it when I was trying to fix the Internal Server Error I had. I t worked for me but maybe I could I have undo the changes before submission.

Reply:
Yes, please undo them.